query_engine/src/datafusion_impl/encoding.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 The CeresDB Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{fmt, sync::Arc};
+
+use datafusion::execution::{runtime_env::RuntimeEnv, FunctionRegistry};
+use prost::bytes::Bytes;
```

Comment:
Replace it with `bytes_ext::Bytes`?

Reply:
It is better actually for reducing the dependency.
I modify the bytes to `bytes = "1"` folling what prost does.